public/assets/sass/style.scss

```diff
@@ -297,6 +321,18 @@ kb-file-tree {
     display: none;
   }
 }
+
+.status-bar-selector {
+  transform: translateY(-24px);
+
+  md-option {
+    height: 2rem;
```

Comment:
for now, we'll keep this as `rem` (changed it from `em`). We'll figure it out later :-)